articles/application-gateway/tutorial-ingress-controller-add-on-existing.md

```diff
@@ -24,6 +24,8 @@ In this tutorial, you learn how to:
 > * Deploy a sample application using AGIC for Ingress on the AKS cluster
 > * Check that the application is reachable through Application Gateway
 
+[!NOTE] When using an AKS cluster with a Windows Server node pool, the recommendation is to use Application Gateway in a different subnet within the same virtual network. [PR#1953](https://github.com/Azure/AKS/issues/1953).
```

Comment:
Is "recommendation" ok since this may be the only solution for now?

Reply:
I find the word "recommendation" misleading - when the only working way is deploying both components in the same vnet, than the wording should reflect that. Everything else will not work.

Reply:
thanks, updated.